extensions/distributed-tracing.md

```diff
@@ -26,10 +26,10 @@ Prometheus are built.
 
 ## Encoding
 
-### In-memory formats
+### Stand-alone event formats
 
 The Distributed Tracing extension uses the key `distributedtracing` for
-in-memory formats
```

Comment:
This may have its origin in when we allowed extensions to deviate from naming conventions to comply with existing contracts. As I understand it, this was supposed to express that the attribute name in event formats is "distributedtracing". As extensions now always have a defined attribute name and MAY have a secondary representation in addition, I suppose this can be dropped.